src/tree/logic-app/LogicAppTreeItem.ts

```diff
@@ -119,10 +119,8 @@ export class LogicAppTreeItem implements IAzureParentTreeItem {
 
     public async update(definition: string): Promise<string> {
         const workflow: Workflow = {
-            definition: JSON.parse(definition),
-            integrationAccount: this.workflow.integrationAccount || undefined,
-            location: this.workflow.location,
-            tags: this.workflow.tags || undefined
+            ...this.workflow,
```

Comment:
Include all workflow properties from previous version, not just `integrationAccount`, `location`, and `tags`.